pkg/apiserver/registry/inventory/vpc_test.go

```diff
@@ -0,0 +1,233 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vpc_inventory_test
+
+import (
+	"fmt"
+
+	logger "github.com/go-logr/logr"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/internalversion"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/client-go/tools/cache"
+
+	runtimev1alpha1 "antrea.io/nephe/apis/runtime/v1alpha1"
+	. "antrea.io/nephe/pkg/apiserver/registry/inventory"
+	"antrea.io/nephe/pkg/controllers/inventory"
+)
+
+var _ = Describe("VPC", func() {
+	var vpcIndexer1 = cache.NewIndexer(
```

Comment:
Can you not init inventory object which iniits the indexers rather than duplicating the code?